<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputerAnswersGame</class>
 <widget class="QWidget" name="ComputerAnswersGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="sendAnswer">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>отправить ответ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberOfBulls">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberOfCows">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>быки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>коровы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="computerAnswer">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
